package/firmware/linux-firmware/amd.mk

```diff
@@ -4,6 +4,11 @@ define Package/amd64-microcode/install
 	$(CP) \
 		$(PKG_BUILD_DIR)/amd-ucode/*.bin \
 		$(1)/lib/firmware/amd-ucode
+	$(INSTALL_DIR) $(1)/boot
+	cat $(PKG_BUILD_DIR)/amd-ucode/microcode_amd*.bin \
+		> $(PKG_BUILD_DIR)/AuthenticAMD.bin
+	echo $(PKG_BUILD_DIR)/AuthenticAMD.bin | \
```

Comment:
Need to review the required directory structure within the cpio archive, perhaps?

Reply:
But doesn’t the bin file need to be in a specific directory within amd-ucode.img?

Reply:
Good catch!, Yes, according to https://wiki.archlinux.org/title/Microcode#Early_loading, yes: "The kernel's early loader expects microcode update files in /kernel/x86/microcode/GenuineIntel.bin or /kernel/x86/microcode/AuthenticAMD.bin inside an uncompressed CPIO archive (initramfs image)."

I think I know what to do...